05_debug/procfs/Makefile

```diff
@@ -0,0 +1,11 @@
+ifneq ($(KERNELRELEASE),)
+# kbuild part of makefile
+obj-m := debug_module.o
+else
+# normal makefile
+KDIR ?= /lib/modules/`uname -r`/build
```

Comment:
uname -r  gives version of kernel on host computer.
It produces wrong results for cross-compilation.

Reply:
I set KDIR to specific kernel before run make.
Do you mean set absolute path in KDIR var?